docs/getting-started/run-applications.md

```diff
@@ -33,12 +33,38 @@ on-demand by Crossplane.
 > published a PostgreSQLInstance with at least one working Composition in order
 > to create the OAM application we'll use in this guide.
 
-## Install the OAM Addon
+## Infrastructure Operator
+
+### Install OAM build-in workloads and traits
 
 As the infrastructure operator our work is almost done - we defined, published,
 and composed the infrastructure  that our application developer and operator
-team-mates will use in the previous guide. One task remains for us, which is to
-install and configure Crossplane's OAM addon.
+team-mates will use in the previous guide. One task remains is, we must define 
```

Comment:
The original "One task remains for us, which is to" is more grammatically correct than the proposed "One task remains is, we must".

Reply:
Incidentally I think the preexisting hyphenation of "team-mate" is old-fashioned/British. "teammate" would be more conventional :)

Reply:
I have no feelings either either way on team-mate/teammate. I think I copied the hypenated example from another document then stuck with that for consistency.